backbone/src/main/java/org/researchstack/backbone/step/active/Recorder.java

```diff
@@ -117,6 +117,15 @@
     @MainThread
     public abstract void stop();
 
+    /**
+     * A force stop will cause this recorder to be immediately cancelled,
+     * the file it was writing will be deleted,
+     *
+     * and no callback will be invoked
+     */
+    @MainThread
+    public abstract void forceStop();
```

Comment:
DataLogger has a method cancel() that sounds like it has the same behavior. Could we name them the same?